src/base/os.lua

```diff
@@ -504,7 +503,7 @@
 				return "chdir " .. path.translate(v)
 			end,
 			copy = function(v)
-				return "xcopy /Q /E /Y /I " .. path.translate(v)
+				return "xcopy /Q /E /Y /I " .. path.translate(v) .. " > nul"

```

Comment:
This really should have been submitted as a separate pull request; I could have merged it while we sorted out the path.normalize() issue. If you want to remove the change above we can let this one go through.